### PR TITLE
Add resolvedEnv for username and password

### PR DESCRIPTION
**Pull request by jessm12**
Signed-off-by: Jess McCreery <jessica.mccreery@ibm.com>

_Provide a description of what has been changed_

### Checklist

- [ ] Commits are signed with Developer Certificate of Origin (DCO)
- [ ] Tests have been added
- [ ] A PR is opened to update the documentation on https://github.com/kedacore/keda-docs
- [ ] Changelog has been updated

Fixes #

### CODE REVIEW

File: pkg/scalers/ibmmq_scaler.go [hunk 1]
```diff
@@ -136,7 +136,17 @@ func parseIBMMQMetadata(config *ScalerConfig) (*IBMMQMetadata, error) {
 		return nil, fmt.Errorf("no username given")
 	}
```
Comment (cpilton) [inline pkg/scalers/ibmmq_scaler.go]: Please remove old code to parse the username on lines 133 - 137

**Review by cpilton — CHANGES_REQUESTED** [review]
The old code to parse the username has been left in. Please remove it

File: pkg/scalers/ibmmq_scaler.go [hunk 2]
```diff
@@ -130,13 +130,17 @@ func parseIBMMQMetadata(config *ScalerConfig) (*IBMMQMetadata, error) {
 		meta.tlsDisabled = defaultTLSDisabled
 	}
 
-	if val, ok := config.AuthParams["username"]; ok {
+	if val, ok := config.AuthParams["username"]; ok && val != "" {
 		meta.username = val
+	} else if val, ok := config.ResolvedEnv["usernameFromEnv"]; ok && val != "" {
+		meta.password = val
```
Comment (zroubalik) [inline pkg/scalers/ibmmq_scaler.go]: `meta.username` imho

Comment (cpilton) [reply pkg/scalers/ibmmq_scaler.go]: Thanks! We've fixed this.

**Review by cpilton — APPROVED** [review]

**Pull request merged**